lib/core/kernel.nit

```diff
@@ -1045,12 +1045,13 @@ universal Char
 	#
 	#     assert 'A'.is_whitespace  == false
 	#     assert ','.is_whitespace  == false
-	#     assert ' '.is_whitespace  == true
+	#     assert ' '.is_whitespace  == true # space
+	#     assert ' '.is_whitespace  == true # non-breaking space
```

Comment:
Since U+00A0 has the `WSpace=Y` property (<https://en.wikipedia.org/wiki/Whitespace_character>), I also agree. In fact, it would be great if the method returned `true` for all `WSpace=Y` code points.